static/src/js/components/Panels/Panels.jsx

```diff
@@ -496,9 +496,7 @@ export class Panels extends PureComponent {
     if (routeWrapper) {
       const routeWrapperWidth = routeWrapper.offsetWidth
 
-      // Set the maxWidth to the available space minus the width of the
-      // map tools.
-      return routeWrapperWidth - 55
+      return routeWrapperWidth - 165
```

Comment:
Maybe add back a comment here so we know why were subtracting. Something simple like this would probably work. 

> Subtracting from the available space to ensure the user menu/login button and map tools remain visible at the maximum panel width